### ISSUE TITLE
invalid backend type

### ISSUE THREAD

**Issue by swizzly** [Thomas-Gelf/icingaweb2-module-extragroups#1]
Hi Tom

Checking the Icingaweb2 logs I found the following messages, I 'don't know whether they really mean any issue, I am just curious what does it mean:

```
Can't get group memberships for user 'myuser' from backend 'Extragroups - rule-based group membership'. An exception was thrown: Icinga\Exception\ConfigurationError in /usr/share/php/Icinga/Authentication/UserGrou
p/UserGroupBackend.php:137 with message: Configuration for user group backend "Extragroups - rule-based group membership" defines an invalid backend type. Backend type "extragroups" is not supported

```

**Comment by Thomas-Gelf** [comment]
Hi @swizzly,

what Icinga Web 2 version are you currently running? Does the `extragroups` module work fine, is the error message new - or has it always been there? I remember a related issue, the log-line was nasty but not a problem. 2.6 should have addressed this issue, however I'm not absolutely sure whether that fix really made it into the latest release.

Cheers,
Thomas

**Comment by swizzly** [comment]
I use: icingaweb2-2.5.3-1.el7.icinga.noarch
The functionality of the extragroups Moduls seems to be ok.

**Comment by Thomas-Gelf** [comment]
That's what I expected, so please ignore the log message for now. We also discovered this issue and as far as I know this should have already been addressed. @lippser: can you confirm this? We talked about this, but i don't remember whether a related fix has already been merged.

**Comment by lippserd** [comment]
Yep, should be fixed with Web 2 2.6.x: See https://github.com/Icinga/icingaweb2/pull/3531 and https://github.com/Icinga/icingaweb2/issues/3500.

**Comment by Thomas-Gelf** [comment]
Great, thank you!
